deps/Makefile

```
@@ -170,6 +174,30 @@ zstd/zstd/lib/libzstd.a:
 zstd: zstd/zstd/lib/libzstd.a
 
 
+PROTOBUF_VERSION := 3.21.12
+PROTOBUF_SRC_DIR := $(PROXYSQL_PATH)/deps/protobuf/protobuf-$(PROTOBUF_VERSION)
+PROTOBUF_INSTALL_DIR := $(PROTOBUF_SRC_DIR)/install
+PROTOBUF_LIB := $(PROTOBUF_INSTALL_DIR)/lib/libprotobuf.a
+
+$(PROTOBUF_LIB):
```

checkmake (0.3.2)

[warning] Target body for "$(PROTOBUF_LIB)" exceeds allowed length of 5 lines (14).

(maxbodylength)